src/styles/fonts.scss

```diff
@@ -0,0 +1,2 @@
+/*Fonts for global styling*/
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
```

Comment:
Please use font-face rules to import fonts like:
```
@font-face {
  font-family: 'Open Sans';
  src: local('Open Sans Bold'), local('OpenSans-Bold'),
  url('/assets/fonts/OpenSans-Bold.woff2') format('woff2'),
  url('/assets/fonts/OpenSans-Bold.woff') format('woff');
  font-weight: bold;
  font-display: swap;
  font-style: normal;
}
```
You have to downloads font files in assets/fonts folder

Reply:
ok, I will do the changes.